static/styles/left_sidebar.css

```diff
@@ -59,7 +59,7 @@ li.show-more-topics {
 .streams_filter_icon {
     float: right;
     opacity: 0.5;
-    font-size: 13px;
+    font-size: 14px;
```

Comment:
Is this the font size of the parent element?  If so, we can potentially just delete this line.